projects/mapit/Makefile

```diff
@@ -1,4 +1,5 @@
 mapit:
+	- $(GOVUK_DOCKER) run $@-app psql -U postgres -c "DROP DATABASE mapit;"
```

Comment:
We should avoid doing things like this, as these steps are meant to be idempotent: we often suggest people [run `make <project>` for troubleshooting](https://github.com/alphagov/govuk-docker#how-to-diagnose-and-troubleshoot), and it could be frustrating if that deletes dev data.

Since the purpose of this is for importing data, could the import scripts do this instead?

Reply:
Ah I see, thanks @benthorner ! I have removed the commit now 👍